components/brave_wallet/browser/filecoin_keyring.cc

```diff
@@ -121,7 +121,7 @@ std::string FilecoinKeyring::EncodePrivateKeyForExport(
     const std::string& address) {
   if (base::Contains(imported_bls_accounts_, address)) {
     return GetExportEncodedJSON(
-        base::Base64Encode(base::make_span(*imported_bls_accounts_[address])),
+        base::Base64Encode(base::span(*imported_bls_accounts_[address])),
```

Comment:
in all these cases we are having double searches, one for `base::Contains`, and then another when calling the subcript `[]` operator. We should be using `imported_bls_accounts_.find` to avoid these double searches.

Reply:
https://github.com/brave/brave-core/pull/26431/commits/6809115eb5276f7e5f97305fc0e0189ade29d515